src/swarm/neo/protocol/connect/NodeConnect.d

```diff
@@ -162,21 +162,29 @@ class NodeConnect
         {
             try
             {
-                enforce(this.e_auth_rejected.set("Empty client name"),          // Validate the client name.
+                // Validate the client name.
+                enforce(this.e_auth_rejected.set("Empty client name"),
```

Comment:
Not related to this change, but there's something really weird in the arguments order of these `enforce` calls.